src/index.html

```diff
@@ -1,12 +1,346 @@
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+              >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="grid-container">
+        <div class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          >
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            Price:
+            <span class="card__price-number">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="card__btn"
+            data-qa="card-hover"
+            >
+            Buy
+          </a>
+        </div>
+
+        <div class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          >
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            Price:
+            <span class="card__price-number">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="card__btn"
+            data-qa="card-hover"
```

Comment:
Use the data-qa attribute only for the first card 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/481d3fe0-e79f-4b23-80ed-f2f1a1406eb7)